app/src/main/java/ir/noori/learningandroid/user/ui/UsersAdapter.kt

```diff
@@ -4,10 +4,11 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import ir.noori.learningandroid.MyApplication
-import ir.noori.learningandroid.user.data.entity.UsersModel
 import ir.noori.learningandroid.databinding.ItemUserBinding
+import ir.noori.learningandroid.user.data.entity.UsersModel
+import javax.inject.Inject
 
-class UsersAdapter(val list : ArrayList<UsersModel>) :
+class UsersAdapter @Inject constructor(val list : ArrayList<UsersModel>) :
```

Comment:
There is no need for the `@Inject` for your adapter.

Reply:
done